package.json

```diff
@@ -238,10 +269,15 @@
     "postshrinkwrap": "node -e \"fs.utimesSync( './node_modules', new Date(), new Date() );\""
   },
   "devDependencies": {
+    "@wordpress/babel-plugin-makepot": "file:vendor/gutenberg/packages/babel-plugin-makepot",
```

Comment:
Isn't `@wordpress/babel-plugin-import-jsx-pragma` the only dev dependency you need in here?

Reply:
You're right, this dependency is not needed at this moment.

Eventually, after we upgrade Calypso's Babel to the latest beta (with `babel.config.js` support, which is very new), I'd like to just import the Gutenberg Babel config into the Calypso config. No more copying the config into the `override` field. Then the `babel-plugin-makepot` will be probably needed? I'm not sure because we'll want to disable it anyway.